sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.net461.cs

```diff
@@ -21,16 +21,16 @@ public enum ServiceVersion
     }
     public partial class UploadFailedEventArgs : Azure.SyncAsyncEventArgs
     {
-        public UploadFailedEventArgs(System.Collections.Generic.List<object> failedLogs, System.Exception exception, bool isRunningSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public UploadFailedEventArgs(System.Collections.Generic.IEnumerable<object> failedLogs, System.Exception exception, bool isRunningSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
         public System.Exception Exception { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<object> FailedLogs { get { throw null; } }
+        public System.Collections.Generic.IEnumerable<object> FailedLogs { get { throw null; } }
```

Comment:
Interesting.. why did we decide to change this? This is essentially an output model so the ReadOnlyList seemed more appropriate to me.

Reply:
will revert